graph-builder/src/main.rs

```diff
@@ -98,12 +98,14 @@ fn main() -> Result<(), Error> {
     let main_state = state;
     HttpServer::new(move || {
         App::new()
+            .wrap(middleware::Compress::default())
             .app_data(actix_web::web::Data::new(main_state.clone()))
             .service(
                 actix_web::web::resource(&format!("{}/v1/graph", app_prefix.clone()))
                     .route(actix_web::web::get().to(graph::index)),
             )
     })
+    .keep_alive(10)
```

Comment:
What does this do?

Reply:
This instructs graph-builder not close the client connections immediately but reuse it for other requests from PE.

Reply:
I trust you to have included this in the commit message ;-)